common/config/rush/pnpm-lock.yaml

```diff
@@ -9152,6 +9152,7 @@ packages:
       integrity: sha512-guDU8PdEdKCVnGxNd1JEkmqukDoc1wodkEqQCWpY1+bX4ZT+ZY520gfVcMeMHYCEO8TAAhScGNke/y7p9qBArA==
       tarball: 'file:projects/testhub.tgz'
     version: 0.0.0
+registry: ''
```

Comment:
Just want to double check this was intentional

Reply:
pnpm/rush did this all on their own, so I think so